ZygorGuidesViewer.lua

```diff
@@ -2800,12 +2913,25 @@ function me:UpdateSkin()
 		self:Print("Load skin Error!")
 	end
 
-	ZygorGuidesViewerFrame_Skipper_PrevButton.ntx:SetTexture(SKINDIR .. "\\titlebuttons")
-	ZygorGuidesViewerFrame_Skipper_PrevButton.ptx:SetTexture(SKINDIR .. "\\titlebuttons")
-	ZygorGuidesViewerFrame_Skipper_PrevButton.htx:SetTexture(SKINDIR .. "\\titlebuttons")
-	ZygorGuidesViewerFrame_Skipper_NextButton.ntx:SetTexture(SKINDIR .. "\\titlebuttons")
-	ZygorGuidesViewerFrame_Skipper_NextButton.ptx:SetTexture(SKINDIR .. "\\titlebuttons")
-	ZygorGuidesViewerFrame_Skipper_NextButton.htx:SetTexture(SKINDIR .. "\\titlebuttons")

```

Comment:
Can you put the code with `LoadSkin(choice)` after this is, and move your code to overwrite the texture (the code with `titlebuttons2`) in the Z_SKINS_HOOKS instead to overwrite the values ? This way we can use the mapping to overwrite buttons as well for all skins